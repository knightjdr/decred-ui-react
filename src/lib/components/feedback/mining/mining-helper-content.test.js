import MiningHelperContent from './mining-helper-content';

describe('MiningHelperContent', () => {
  test('Returns content', () => {
    const wrapper = MiningHelperContent('#000', '#000', '#000', 40, 1.5);
    expect(wrapper).toMatchSnapshot();
  });
});
