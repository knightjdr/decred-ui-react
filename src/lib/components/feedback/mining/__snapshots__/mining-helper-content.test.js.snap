// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiningHelperContent Returns content 1`] = `
<svg
  className="decred-ui-mining-svg"
  height={40}
  viewBox="3 0 55 48"
  width={44}
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    className="decred-ui-mining-pickaxe"
    style={
      Object {
        "animationDuration": "1.5s",
      }
    }
  >
    <path
      d="M43.896,17.517c0.458-0.866,0.458-1.901,0-2.768c-0.765-1.445-2.557-1.999-4.001-1.233l0,0C41.382,14.685,42.726,16.029,43.896,17.517z"
      fill="#000"
    />
    <path
      d="M32.991,20.338l-23.13,23.11c-1.154,1.16-1.149,3.037,0.011,4.192c1.16,1.153,3.037,1.149,4.191-0.01L37.114,24.48L32.991,20.338z"
      fill="#000"
    />
    <path
      d="M41.315,24.48l7.684,7.704c-0.009-5.326-1.812-10.494-5.122-14.667l0,0c-1.17-1.488-2.515-2.832-4.001-4.001l0,0c-4.159-3.283-9.309-5.061-14.605-5.042l0,0l7.724,7.664l0.939,1.06l-2,2.102l1.061,1.041l4.122,4.142l1.141,1.081l2-2.102L41.315,24.48z"
      fill="#000"
    />
  </g>
  <g
    className="decred-ui-mining-blocks"
    style={
      Object {
        "animationDuration": "1.5s",
      }
    }
  >
    <rect
      fill="#000"
      height="5.624"
      transform="matrix(0.6192 0.7852 -0.7852 0.6192 55.779 -23.5726)"
      width="2"
      x="51.194"
      y="42.911"
    />
    <rect
      fill="#000"
      height="4.813"
      transform="matrix(0.6159 -0.7878 0.7878 0.6159 -20.6488 48.1631)"
      width="1.999"
      x="38.069"
      y="42.851"
    />
    <rect
      fill="#000"
      height="4.475"
      transform="matrix(0.8088 0.5881 -0.5881 0.8088 33.9183 -23.1469)"
      width="1.999"
      x="51.559"
      y="38.354"
    />
    <rect
      fill="#000"
      height="5.624"
      transform="matrix(0.7287 -0.6848 0.6848 0.7287 -19.0036 43.4761)"
      width="2.001"
      x="44.374"
      y="42.912"
    />
  </g>
</svg>
`;
